taxi/views.py

```diff
@@ -64,13 +65,13 @@
 
 class CarCreateView(LoginRequiredMixin, generic.CreateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
     success_url = reverse_lazy("taxi:car-list")
 
 
 class CarUpdateView(LoginRequiredMixin, generic.UpdateView):
     model = Car
-    fields = "__all__"
+    form_class = CarForm
```

Comment:
The `CarUpdateView` also uses `CarForm`, which is correct for updating car data. Verify that `CarForm` supports updates as expected.